raster/r.topidx/r.topidx.html

```diff
@@ -1,24 +1,30 @@
 <h2>DESCRIPTION</h2>
 
-<em>r.topidx</em> creates topographic index (wetness index), ln(a/tan(beta)), map from
-elevation map
-<p>where
+<em>r.topidx</em> creates topographic index (wetness index), ln(a/tan(beta)),
+map from elevation map where
 <dl>
-<dd>a: the area of the hillslope per unit contour length that drains through any point,
-<p><dd>tan(beta): the local surface topographic slope (delta vertical) / (delta horizontal).
+<dt>a</dt>
+  <dd>the area of the hillslope per unit contour length that drains through any
+      point,</dd>
+<dt>tan(beta)</dt>
+  <dd>the local surface topographic slope (delta vertical) / (delta
+      horizontal).</dd>
 </dl>
-<p>Input maps may have NULL values.  For example, if you have a MASK for
-a watershed (basin map from <em>r.water.outlet</em>), the
-following command will create a masked elevation map (belev):
+<p>Input maps may have NULL values.  For example, if you have a MASK for a
+watershed (basin map from <em>r.water.outlet</em>), the following command will
+create a masked elevation map (belev):
 <div class="code"><pre>
 r.mapcalc "belev = if(isnull(basin), basin, elev)"
 </pre></div>
 <p>
 <em>r.stats -Anc</em> prints out averaged statistics for topographic index.
 
+<p>Unprojected lat/long locations are not supported.
```

Comment:
So what would I, user, need to do then?

Reply:
They need to create a new projection location (project now) and import data into that project. I can add this info, but there are more modules that don't support lat/long projections.

Reply:
Also, I'm not sure how much detail we want to add to individual module manuals. I think this reprojection info should be more global. Anyway, I've added more info.

Reply:
Y'a I know a manual isn't the place for a tutorial... But it's also the only place you go when you want to use a module. Something simple like: (the thing) must be (in a) projected (the good term)...